generator/src/components/Button/index.svelte

```diff
@@ -1,17 +1,24 @@
 <script lang="ts">
-  export let onClick: () => void
+  export let onClick: (() => void) | null = null
```

Comment:
`null` is bad, use `undefined`
```suggestion
  export let onClick: (() => void) | undefined = undefined
```